core/src/main/java/io/cucumber/core/feature/FeaturePath.java

```diff
@@ -101,4 +111,13 @@ private static String normalize(final String value) {
         return value.toLowerCase(Locale.US).replaceAll("[^a-z0-9]+", "");
     }
 
+    private static void warnWhenFileSystemPathToResources(String featureIdentifier) {
+        Matcher matcher = FILESYSTEM_PATH_TO_RESOURCES.matcher(featureIdentifier);
+        if (matcher.matches()) {
+            log.warn(() -> String.format("Please replace feature path '%s' with the classpath '%s' to avoid ambiguity.",
+                featureIdentifier,
+                matcher.replaceAll("classpath:$1")));
```

Comment:
I think we're missing a bit of explanation here. Some one reading this might wonder:

* What did I do wrong?
* Why should I follow this suggestion, my code works right?